gateway/api/middleware.go

```diff
@@ -315,10 +316,26 @@ func (a *Api) TrackRequest(eventName string) func(c *gin.Context) {
 			c.Next()
 			return
 		}
+		org, err := pgorgs.New().FetchOrgByContext(ctx)
+		if err != nil {
+			log.Errorf("failed fetching org, reason=%v", err)
+		}
```

Comment:
Make sure to return after an error, otherwise it will blow up with nil pointer exception